WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -471,29 +437,32 @@ private extension CommentDetailViewController {
         }
     }
 
-    func configureContentCell(_ cell: CommentContentTableViewCell, comment: Comment) {
-        cell.configure(with: comment) { [weak self] _ in
-            self?.tableView.performBatchUpdates({})
+    func configureContentCell(_ cell: CommentDetailContentTableViewCell, comment: Comment) {
+        let onOptionSelected: (CommentDetailContentTableViewCell.MenuOption) -> Void = { [weak self] option in
+            guard let self else {
+                return
+            }
+            switch option {
+            case .userInfo:
+                presentUserInfoSheet()
+            case .editComment:
+                editButtonTapped()
+            case .share:
+                shareCommentURL()
+            case .changeStatus(let status):
+                print("Option \(status) tapped")
+            }
         }
-
-        cell.contentLinkTapAction = { [weak self] url in
+        let contentConfig = CommentDetailContentTableViewCell.ContentConfiguration(comment: comment) { [weak self] _ in
+            UIView.performWithoutAnimation {
+                self?.tableView.performBatchUpdates({})
+            }
```

Comment:
When rendering the comment, we use the `performBatchUpdates` method to relayout the cells and update their heights. This approach was part of the previous implementation. What I've added is the wrapping of this method inside `UIView.performWithoutAnimation`, ensuring that the cells' relayout occurs without any animation. In my opinion, the default animation appeared clunky.